assets/dev/scss/editor/panel/controls/_code.scss

```diff
@@ -0,0 +1,39 @@
+.elementor-control-type-code {
+
+	&.elementor-control-dynamic-value {
```

Comment:
Try to share the code with the textarea control.

Reply:
Done.